Userland/Libraries/LibJS/Runtime/Temporal/Calendar.cpp

```diff
@@ -370,11 +370,13 @@ Object* get_temporal_calendar_with_iso_default(GlobalObject& global_object, Obje
         return &static_cast<PlainDate&>(item).calendar();
     if (is<PlainDateTime>(item))
         return &static_cast<PlainDateTime&>(item).calendar();
+    // TODO: PlainMonthDay
     if (is<PlainTime>(item))
         return &static_cast<PlainTime&>(item).calendar();
+    if (is<PlainYearMonth>(item))
+        return &static_cast<PlainYearMonth&>(item).calendar();
```

Comment:
Same in the `LibJS: Handle ZonedDateTime in GetTemporalCalendarWithISODefault` commit as above